doc/api/url.md

```diff
@@ -909,7 +909,7 @@ It performs the inverse operation to [`url.domainToUnicode()`][].
 ```js
 const url = require('url');
 console.log(url.domainToASCII('español.com'));
-// Prints xn--espaol-zwa.com
+// Prints xn--español-zwa.com
```

Comment:
The original was the correct [Punycode](https://tools.ietf.org/html/rfc5891#section-4.4) encoded result. 
```
> url.domainToASCII('español.com')
'xn--espaol-zwa.com'
>
```
and its inverse
```
> url.domainToUnicode('xn--espaol-zwa.com')
'español.com'
>
```
whereas:
```
> url.domainToUnicode('xn--español-zwa.com')
''
>
```

Reply:
my bad. i got it now.
closing this pr. thanks.